catchup/catchpointService.go

```diff
@@ -466,6 +466,7 @@ func (cs *CatchpointCatchupService) processStageBlocksDownload() (err error) {
 				}
 				if attemptsCount <= uint64(cs.config.CatchupBlockDownloadRetryAttempts) {
 					// try again.
+					cs.log.Infof("Failed to download block %d. %v", topBlock.Round()-basics.Round(blocksFetched), err)
```

Comment:
Perhaps show attempt x of y in message    
ie:
Failed to download block %d (attempt %d of %d). %v

Reply:
I don't know if that's a super useful piece of information here, but I don't mind adding it either.